tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+import fakeredis
+import pytest
+
+
+@pytest.fixture
+def mock_redis_server():
+    return fakeredis.FakeServer()
+
+
+@pytest.fixture
+def mock_redis_pubsub(mocker):
+    mock_pubsub = mocker.Mock()
+
+    mocker.patch(
+        "funcx_web_service.routes.funcx.g_redis_pubsub", return_value=mock_pubsub
```

Comment:
Like you pointed out, let's consider replacing this to have redis_pubsub use fakeredis when we have tests for that component. Probably that should go into funcx_common.

Reply:
I want `funcx-common` to provide something like `FUNCX_COMMON_REDIS_BACKEND=fakeredis` to automatically take care of this, so we just need to set an env var (or, equivalently, pass some simple flag) to get `fakeredis` stubbed _everywhere_.
But it's future work -- I didn't want to let this PR get too big.

Reply:
:+1:  I think this PR is good to go